### ISSUE TITLE
Support BED input?

### ISSUE THREAD

**Issue by PanZiwei** [dohlee/metheor#21]
Hi,
Thanks for the amazing tool. I already have per-site methylation calling results from long-read sequencing in BED format with chromosome, start position, end position, and methylation frequency for each CpG site. 

Is it possible to use the BED file as `metheor` input  for methylation heterogeneity measures analysis? Any hints for the implementation? Thanks.

**Comment by little-spice** [comment]
The BED file lacks information about reads, thus I believe it cannot be utilized as input for calculating heterogeneity.

**Comment by dohlee** [comment]
Yes, metheor is designed to deal with CpG methylation patterns extracted from sequencing reads, so BED file would not be the best format for that.
